bugsnag-plugin-android-ndk/src/main/assets/include/bugsnag.h

```diff
@@ -36,6 +39,12 @@ void bugsnag_set_user_env(JNIEnv *env, char* id, char* email, char* name);
 void bugsnag_leave_breadcrumb(char *message, bsg_breadcrumb_t type);
 void bugsnag_leave_breadcrumb_env(JNIEnv *env, char *message, bsg_breadcrumb_t type);
 
+void bugsnag_add_on_error(bsg_on_error on_error);
+void bugsnag_remove_on_error(bsg_on_error on_error);
```

Comment:
If the argument is not going to be used, there's no need to require it and it seems a bit confusing to include it.